react/rules/ITA.js

```diff
@@ -0,0 +1,72 @@
+import { isPastDate } from '../utils/dateRules'
+
+export default {
+  country: 'ITA',
+  personalFields: [
+    {
+      name: 'firstName',
+      maxLength: 100,
+      label: 'firstName',
+      required: true,
+    },
+    {
+      name: 'lastName',
+      maxLength: 100,
+      label: 'lastName',
+      required: true,
+    },
+    {
+      name: 'email',
+      maxLength: 100,
+      label: 'email',
+      hidden: true,
+    },
+    {
+      name: 'document',
+      maxLength: 50,
+      label: 'document',
+      required: false,
+      hidden: true,
+    },
+    {
+      name: 'homePhone',
+      maxLength: 30,
+      label: 'homePhone',
+    },
+    {
+      name: 'gender',
+      maxLength: 30,
+      label: 'gender',
+    },
+    {
+      name: 'birthDate',
+      maxLength: 30,
+      label: 'birthDate',
+      type: 'date',
+      validate: isPastDate,
+    },
+  ],
+  businessFields: [
+    {
+      name: 'corporateName',
+      maxLength: 100,
+      label: 'corporateName',
+    },
+    {
+      name: 'tradeName',
+      maxLength: 100,
+      label: 'tradeName',
+    },
+    {
+      name: 'corporateDocument',
+      maxLength: 30,
+      label: 'corporateDocument',
+      required: false,
```

Comment:
Curiosity: Why are some of the fields explicitly `required: false` and others aren't?

Reply:
To override the default rules